template/module/README.rst

```diff
@@ -65,8 +65,8 @@ Images
 Contributors
 ------------
 
-* Firstname Lastname <email.address@example.org>
-* Second Person <second.person@example.org>
+* Firstname Lastname <email.address@example.org> (optional company website url)
```

Comment:
We already have a [section for funders](https://github.com/OCA/maintainer-tools/blob/d33158faa1b762a7428a446a6f9711c419546547/template/module/README.rst#funders).

Often contributors are individuals paid by their company so putting an URL to the company web site next to contributor emails sounds reasonable.